examples/LottieExtension.cpp

```diff
@@ -125,14 +140,56 @@ struct UserExample : tvgexam::Example
o2BGt4LpxYnb+ngdAeAtTGAUzuxSOKOoZKgsO1hivAAAtBgTBgpmdimcXpQMJ/4TIQCcjDEKZ3Qp6C/KDowrGSyr1nnnAQAZM1LY36VgdjdX/FMgAEzBBEbhnB4FfQXFh8flSnHeLQEATsB0hYrm9coU23PUfto6c+ijB9MVqXBan8KZ3Xm3AgB4i3B6lwqn9nPyrwF3AGoUzu6W6Q6UHByTszwSAIA8mcAonNc7MbIfNeEOgIegt6jotL6OfXcUAJqBCQNFp/Zx8vfEGcyTKYSKTu2TKfARAkDWTDGcuBDjlr83zl51MFGg6LR+mSIfIwBkxRR/fgHGXdi68OnVyQRG0Sl8EQEgCyYKFC2cxit+KeCslQITBhMhIOQLCQCNYgIzcfLngisVfIopMYVA0fxpYoJpAGiMaOE0nvmniACQItMTKZzdlXcbANB2wtk9Mt28uZ4mAkDKwhndMr18SQEgLaYnUjizmHcbbYcAkDYjRXOn8ckCQBqMUTS/VzxeTR+nqQYwkVE4qyfvNgCg5YWzu2VCTlWNwKfaIOGMLpkuBqsAgC9TDBXOYFxVoxAAGiiazV0AAPAVzeU3tJEIAA1keiKZbu4CAECtTHfEqP8GIwA0GMsHA0Dt+O1sPAJAgwW9BdYKAIAamGKogNepG44zUwaCfgaxAEC1gn7e+c8CASADQR9fZgCoFr+Z2SAAZMCERkFvIe82AKDpBb0FFlbLCAEgI2YaAQAAphLwW5kZAkBGAl4HBIApmR4G/2WFAJARUwhZwxoAJmEKAb+TGeKTzhCTWgDAyfEbmS0CQIZYGwAATs4U+Y3MEgEgQ6bAxw0AJxMUCABZ4oyUIdItAEyCWVMzxaedIRMGEq+3AsDxzM9/I5EZPu0sGckYEgAAvJUJDBdIGSMAZI2ECwDHCzj7Z42zUdb4xAHgeNwdzRwvXWYs6O+SemzebQBAc2ECoMwRADIWTmeVKwBA/ohcAAB0oMkDgA24Vw0AQCtKXDLZnycNAE7xSLrdAACALNgoGJ7s75MGgEIhnLQYAAA0p6KN/QOAbJkAAABAK6oU6ggApS4CAAAArWjGoH8AMBc8MyrpSKoNAQCARttvztpammyDqV8DdNqSWjsAAKDxjF6capMq5gFwBAAAAFrLlOfuKQOACQwBAACAFmI09bl7ygBgrd2QTjsAACALVm79VNtMGQCiOHpM0qQDCQAAQNMYjXqGn5pqo6kfAVzwzKjknk6nJwAA0FDGPDbVGwBSlYsBOWMerL8jAADQaMa6h6rZrroA4Mx36msHAABkIQ7NA9VsZ6rdYWXTmheN3Dn+LQEAgEYyTpvCVRtWVbNtVXcAJClw7g7/lgAAQAZuq3bD6gOA1f+T5LzaAQAAjWYD2buq3bjqAGDWbNgh6fteLQEAgEa7z6zatKfajasOAJLkjP6q9n4AAECj2aC2c3RNAaCwYsMjkh6tqSMAANBo3yku37CuloKaAoAkOWc+W2sNAABoHGddzefmmgNAYdX670n6bq11AACgIb5eWL3x8VqLag4AkhRa90difQAAAPI2Glr3n3wKvQKAWb3xZSf3OZ9aAACQDmPMZ8zqjdt9ar0CgCRFlfC/yWmrbz0AAPBnnDYFZfPX3vX1HLz07JoVoXVPSeqpZz8AAKAmx8LAvNMsX/+C7w687wBIUtfy9ZuNMZ+sZx8AAKBGTr9fz8lfqvMOwGviTavvkXRDGvsCAACTMV+OVq7/N/Xupa47AK8JR7p/S1LNryAAAICaPBj2Dv1hGjtK5Q6AJLkX3jknrlR+ZJzekdY+AQDA69aFUflqc97zx9LYWSp3ACTJvP0nRyITvVfSrrT2CQAAJCe9FCa6Lq2Tv5RiAJAks/zpXaF1VzrppTT3CwBApzIyz0XOXmPWbDiU5n5TDQCSZFZv3B4VC5dKeirtfQMA0FGMHg4q5vJalvmtftcN4l48pz8pT7tXzr2nUccAAKCN3RX2Dv+OOWtrQ6beb1gAkCTnZOyzqz/pnD4rKWzksQAAaBMlI/cnwYqNf2uMXKMO0tAA8JrK5tXvMk53SDo1i+MBANCidlgFNxVXPvPjRh8o9TEAJ1JYseGRUMlqSbdJjUszAAC0qNjJfCHsGl2exclfyugOwBtVNq25IpD7opOWZ31sAACa0Dpr3e8XV298OsuDZh4AJMmtW1uwRfdvnXOflLQojx4AAMiTk14wxn02XL7xjkY+6z+ZXALAa9y6tYWkYD/sZD5l5M7JsxcAALJgpGed9Llwy5lfNTfek+TYR/7c3R8Kk3N+dp2kj0nuekndefcEAEBqjIZk3ded01eiVRsfyeOK//iWmozbsGpmEugGmeADcvZKyfTn3RMAAB4OybiH5Mw3w5HufzKXPjmWd0Nv1HQB4I3cQ1dFldlDF4TOXeWMLnYy5xq5MyQV8u4NAIA3GDdOW5wxLxnpiTjQg8Xz1z/bDFf6J9PUAeBE3ENXRZo/tDSu6G0msNMlzTQumCbZXuvM9Lz7AwC0r8C4Ickcc07HFLhh54IjkbWvaNXGncbI5t0fAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMJX/DzPmxrt/f+ISAAAAAElFTkSuQmCC","e":1}})";
```

Comment:
Let's remove this sample. The slot sample requires image overriding, no need to test base64 data.